packages/-ember-data/tests/unit/adapters/rest-adapter/fetch-options-test.js

```diff
@@ -0,0 +1,143 @@
+import { setupTest } from 'ember-qunit';
+import { module, test } from 'qunit';
+import { fetchOptions } from '@ember-data/adapter/rest';
+
+module('unit/adapters/rest-adapter/fetch-options', function(hooks) {
+  setupTest(hooks);
```

Comment:
I _think_ we can remove this part? It seems like all the tests are using the utility function directly, and don't need any additional test setup. Did I miss something?

```suggestion
```